src/noframe.js

```diff
@@ -0,0 +1,40 @@
+export default function noframe(target, container) {
+  let els = typeof target === 'string' ? document.querySelectorAll(target) : target;
+  if (!('length' in els)) {
+    els = [els];
+  }
+  for (let i = 0; i < els.length; i++) {
+    const el = els[i];
+    const height = el.offsetHeight;
+    const width = el.offsetWidth;
+    if (typeof container !== 'undefined' && container !== 'false') {
```

Comment:
This is a code smell, checking for a string `'false'` when it should be used as a boolean in some way.